src/components/component-playground.jsx

```diff
@@ -554,48 +558,44 @@ module.exports = React.createClass({
     });
   },
 
-  _getComponentsWithMatchingFixtures() {
-    return _.pick(_.mapValues(this.props.components, function(details) {
-      return _.assign({}, details, {
-      // Only select fixtures matching search text and rebuild the object.
-        fixtures: _.pick(details.fixtures,
-            function(fixtureProps, fixtureName) {
-              return fixtureName.indexOf(this.state.searchText) !== -1;
-            }.bind(this)
-        )
-      });
-    }.bind(this)),
+  _getFilteredComponents() {
+    if (this.state.searchText.length < 2) {
+      return this.props.components;
+    }
+
+    // Get components which match search text by one or more fixture names, only
+    // keeping those fixtures. Also keep the selected fixture.
+    return _.pick(_.mapValues(this.props.components,
+        function(details, componentName) {
```

Comment:
The problem here is that, as it was suggested in a previous PR, `component-playground.jsx` does not have any ES6 features, so the style would get out of sync. We can create a new issue for adding some ES6 features, if everybody agrees.